packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1119,6 +1121,46 @@ module.exports = function (log, config, bounces) {
     });
   };
 
+  Mailer.prototype.passwordForgotOtpEmail = async function (message) {
+    log.trace('mailer.passwordForgotOtpEmail', {
+      email: message.email,
+      uid: message.uid,
+    });
+
+    const templateName = 'passwordForgotOtp';
+    const code = message.code;
+    const links = this._generateLinks(
+      this.verificationUrl,
```

Comment:
Don't think we need `verificationUrl` since that might try to create a password reset link? Should be able to pass in null.